application/services/telegram.py

```diff
@@ -0,0 +1,90 @@
+import requests
+import logging
+
+from telegram import Update
+from telegram.ext import ApplicationBuilder, ContextTypes, CommandHandler, filters
+
+from application.services import common
+from application.services import security as vault
+from application.services import sms_service
+
+logging.basicConfig(
+    format='%(asctime)s - %(name)s - %(levelname)s - %(message)s',
+    level=logging.INFO
+)
```

Comment:
This should probably be moved to driver.py eventually, but this works for now :D